MAUI/AIAssistView/styles.md

```diff
@@ -952,5 +952,87 @@ To style the send item button based on its state, set values to the in-built key
         ....
     }
 
+{% endhighlight %}
+{% endtabs %}
+
+## Stop responding view styling
+
+To style the stop responding view based on its appearance, set values to the in-built keys in the resource dictionary.
+
+<table> 
+<tr> 
+<th> Key </th>
+<th> Description </th> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingIconColor </td>
+<td> Icon color for the stop responding action. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingTextColor </td> 
+<td> Text color for the stop responding action. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingFontFamily </td> 
+<td> Font family used for the stop responding text. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingFontSize </td> 
+<td> Font size of the stop responding text. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingFontAttributes </td> 
+<td> Font attributes used for the stop responding text. </td>
+</tr>
+<tr> 
+<td> SfAIAssistViewStopRespondingStroke </td> 
+<td> Stroke color for the stop responding action. </td> 
```

Comment:
Stroke color for the stop responding view.

Reply:
updated